voting-machine/host/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "digital-signature-cli"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+env_logger = "0.9.0"
+log = "0.4.17"
+clap = "3.1.18"
+ctor = "0.1"
+tempfile = "3.3"
+voting-machine-methods = { path = "../methods" }
+voting-machine-core = { path = "../core" }
+risc0-zkvm-core = "0.7"
+risc0-zkvm-host = "0.7"
+risc0-zkvm-serde = "0.7"
```

Comment:
Please keep these sorted

Reply:
In what manner? I just googled and found two different strategies - sorted alphabetically and sorted alphabetically within "class" of dependency.